### ISSUE TITLE
Accessing worstcaseplot and contourplot

### ISSUE THREAD

**Issue by chadhazlett** [carloscinelli/sensemakr#14]
We had envisioned these being either called via plot.sensemade(type=), or directly by the user. 
The latter is not generally working -- special quantities are computed in plot.sensemade, such as benchmarks$benchmarks_2plot1, which are not present if user calls directly.  For now I'm removing the "@export" on these, but we should decide if we want them to be user facing, and clean up as needed if we do.

**Comment by statsccpr** [comment]
I see, that's exactly right.

The topmost generic plot.sensemade has the portion that takes 

the showvars arguments into
subsets quantities
assigns into

`benchmarks$benchmarks_2plot1`
`benchmarks$benchmarks_2plot2`

then dispatches to
contourplot()
worstcaseplot()

contourplot and worstcase plot will then only display

`benchmarks$benchmarks_2plot1`
`benchmarks$benchmarks_2plot2`

depending on the decision moving forward we can tuck in this showvars assignment step into these two specific methods if desired

**Comment by chadhazlett** [comment]
Thanks Michael. I'm open to what you guys think on this.  At the moment it
would seem expeditious to require the user to go through plot().

**Comment by statsccpr** [comment]
While adding examples and parameter documentations to the generic plot() and nonexported worstcaseplot() contourplot(), convinced me that we should do one or the other.

1. Only rely on the generic and do not export worstcase + contour
or
2. get rid of the generic entirely, and rely on worstcase() + contourplot()

I think the generic plot() in concept, is nice since it forces the default plot to be the contour plot.
But that's about it.
the generic plot() doesn't save typing at all, if you want to get the worstcase option
plot(worstcase) is actually more than the direct worstcaseplot()

So, I'm actually thinking the generic plot() is not really worthwhile. Especially since there are custom arguments that do not really align with the vanilla arguments in plot(),

Long story short, I think I'm leaning towards casting my vote to abandoning generic plot() while exporting and relying worstcaseplot() and contourplot()

I can lean the other direction too, but definately do not think we should support both

A negative about using generic s3 plot method is that looking up documentation is hard for someone not knowing where to look

the user needs to know to look at
?plot.sensemakr

since ?plot is not helpful

This would be in support of exporting ?countourplot or ?worstcaseplot
since they can can have their own distinguished help docs

thoughts?

**Comment by statsccpr** [comment]
to try out both options, both of the lower level plots are directly accessible now, can also be exported

https://github.com/chadhazlett/sensemakr/commit/51422957188f5e512be1f92f1726cf9ad4708baa

So currently, can use all 3: generic, worstcase, contour

**Comment by statsccpr** [comment]
Closing issue for now, all 3 work, generic plot() + worstcase() + contour()
all 3 share the same single documentation
?plot.sensemakr
?worstcaseplot
?contourplot

can revisit if we do not want all 3 available
